formats/developers/ome-units.txt

```diff
@@ -0,0 +1,253 @@
+The OME System of Units
+=======================
+
+.. topic:: Overview
+
+   This sections explains the OME system of Units.
+
+Implementation
+^^^^^^^^^^^^^^
+
+The `ome.xsd` schema file defines new type enumerations: UnitsLength,
+UnitsTime, UnitsPressure, UnitsAngle, UnitsTemperature,
+UnitsElectricPotential, UnitsPower and UnitsFrequency. These types are used
+throughout the model. Each new unit attribute is in a pair with an existing
+attribute, e.g. PinholeSize and PinholeSizeUnit. The Xsd-Fu application then
+processes these into java classes that use types based on the new
+`ome.units.*` classes.
+
+The classes in `ome.units` are inspired by but not a direct implementation of
+the specification for a Units of Measurement API available from
+http://www.unitsofmeasurement.org.
+
+The classes have been constructed in a way that means they could in future be
```

Comment:
"These classes have been constructed to allow for future wrapping of the Units"... ?  Makes it a bit shorter.